README.nl.md

```diff
@@ -0,0 +1,104 @@
+[日本][JA],
+[正體中文][ZH_TW],
+[简体中文][ZH_CN],
+[한국어][KO_KR]
+
+[AR]:README.ar.md
+[BD_BN]:README.bd_bn.md
+[CA]:README.ca.md
+[DE]:README.de.md
+[EN]:README.md
+[ES]:README.es.md
+[FA]:README.fa.md
+[FR]:README.fr.md
+[HI_IN]:README.hi_in.md
+[ID]:README.id.md
+[IT]:README.it.md
+[JA]:README.ja.md
+[KO_KR]:README.ko_kr.md
+[MM]:README.mm.md
+[PL]:README.pl.md
+[PT_BR]:README.pt_br.md
+[RO]:README.ro.md
+[RU]:README.ru.md
+[TR]:README.tr.md
+[VI]:README.vi.md
+[ZH_CN]:README.zh_cn.md
+[ZH_TW]:README.zh_tw.md
+[NL]:README.nl.md
+
+Originele broncode voor de Apollo 11 besturingscomputer (AGC). Broncode voor de besturingsmodule (Comanche055) en maanmodule (Luminary099). Gedigitalizeerd door [Virtual GC][3] en het [MIT Museum][4]. Het doel is om een repository te zijn van de oorspronkelijke Apollo 11 broncode en dus zijn PRs welkom voor elk probleem in de overgenomen broncode van [Luminary 099][5] en [Comanche 055][6] en elk bestand dat ik vergeten ben.
+
+## Bijdragen
+Lees alsjeblieft [CONTRIBUTING.nl.md][7] voordat je een pull request opent.
+
+## Compileren
+Als je geinteresseerd bent in het compileren van de originele broncode bekijk dan [Virtual AGC][8].
+
+## Toebedeling
+
+&nbsp;         | &nbsp;
+:------------- | :-----
+Copyright      | Publiek domein
+Comanche055    | Onderdeel van de broncode voor Colossus 2A, de Apollo besturingscomputer (AGC) van de besturingsmodule (CM) voor Apollo 11<br>`Assemble revision 055 of AGC program Comanche by NASA`<br>`2021113-051. 10:28 APR. 1, 1969`
+Luminary099    | Onderdeel van de broncode voor Luminary 1A, de Apollo besturingscomputer (AGC) van de maanmodule (LM) voor Apollo 11<br>`Assemble revision 001 of AGC program LYM99 by NASA`<br>`2021112-061. 16:27 JUL. 14, 1969`
+Samensteller   | yaYUL
+Contact        | Ron Burkey <info@sandroid.org>
+Website        | www.ibiblio.org/apollo
+Digitalizering | Deze broncode is overgeschreven of anderszins overgenomen van gedigitalizeerde afbeeldingen van een boek van het MIT Museum. Deze digitalizering is uitgevoerd door Paul Fjeld en geregeld door Deborah Douglas van het voorgenoemde museum. Hartelijk dank aan beiden.
+
+### Contract en Goedkeuring 
+*Afgeleid van [CONTRACT_AND_APPROVALS.agc]*
+
+Dit AGC programma zal worden benoemd als Colossus 2A.
+
+Dit programma is bedoeld voor gebruik in de CM zoals vastgesteld in rapport `R-577`. Dit programma is voorbereid onder DSR project `55-23870`, gesponsord door het Manned Spacecraft Center van de National Aeronautics and Space Administration via contract `NAS 9-4065` met het Instrumentation Laboratory, Massachusetts Instute of Technology, Cambridge, Mass.
+
+Ingediend door        | Rol  | Datum
+:-------------------- | :--- | :---
+Margaret H. Hamilton  | Colossus Programming Leader<br>Apollo Guidance and Navigation | 28 Mar 69
+
+Goedgekeurd door   | Rol  | Datum
+:----------------- | :--- | :---
+Daniel J. Lickly   | Director, Mission Program Development<br>Apollo Guidance and Navigation Program | 28 Mar 69
```

Comment:
Possibly translate `Director`, `[Colossus] Project Manager` and `Deputy Director`

Reply:
I'll have to ask around about this one, I'm uncertain if the Dutch terms for these positions carry the same meaning I'll update this after!

Reply:
I've translated those that have direct Dutch translations and left the one (Programming Leader) which has no direct translation. If you want me to be a bit more liberal and translate that one as well let me know!